runtime/src/main/java/org/corfudb/runtime/CorfuRuntime.java

```diff
@@ -476,6 +476,9 @@
          */
         @Default
         int prometheusMetricsPort = MetricsUtils.NO_METRICS_PORT;
+
+        @Default
+        String codecType = "NONE";
```

Comment:
This will be exposed to the applications, a string is more flexible than our Codec.Type, just in case we need to refactor, I don't want to break applications code.